atlasdb-timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/PaxosTimeLockServerIntegrationTest.java

```diff
@@ -143,6 +148,51 @@ public void timestampServiceShouldRespectDistinctClientsWhenIssuingTimestamps()
     }
 
     @Test
+    public void timestampServiceRespectsTimestampManagementService() {
+        TimestampService timestampService = getTimestampService(CLIENT_1);
```

Comment:
Actually the timestamp service (and management service) for client 1 can be private final fields of the class, so I've made them that.